data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/handler/ProbeRequestUtil.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.knative.eventing.kafka.broker.receiver.impl.handler;
+
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.http.HttpServerRequest;
+
+public class ProbeRequestUtil {
+
+  public static final String PROBE_HEADER_NAME = "K-Network-Probe";
+
+  public static final String PROBE_HEADER_VALUE = "probe";
+
+  public static final String PROBE_HASH_HEADER_NAME = "K-Network-Hash";
```

Comment:
I do like the docs from here: https://github.com/knative/pkg/blob/b558677ab03404ed118ba3e179a7438fe2d8509a/network/network.go#L38-L51

perhaps either copy, or better link to those?

Reply:
+1